frontend/lib/src/components/core/Block/ElementNodeRenderer.tsx

```diff
@@ -221,6 +221,7 @@ const RawElementNodeRenderer = (
     widgetMgr: props.widgetMgr,
     disabled: props.widgetsDisabled,
     fragmentId: node.fragmentId,
+    elementHash: node.elementHash,
```

Comment:
Can we move this into elementProps? Since I believe it could benefit all of our elements

Reply:
I moved the memo to the `ElementNodeRenderer` and now the `elementHash` is already consumed before we land here, so I have removed it from this point again